libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -685,33 +695,36 @@ static CK_RV prvRsaKeyAttParse( const CK_ATTRIBUTE * pxAttribute,
     CK_RV xResult = CKR_OK;
     int32_t lMbedTLSResult = 0;
     CK_BBOOL xBool;
-    mbedtls_rsa_context * pxRsaContext = ( mbedtls_rsa_context * ) pxMbedContext->pk_ctx;
 
-    configASSERT( pxRsaContext != NULL );
+    configASSERT( ( mbedtls_rsa_context * ) pxMbedContext->pk_ctx != NULL );
```

Comment:
What's the purpose of this change? I think the previous version is more readable for the NULL check with `configASSERT( pxRsaContext != NULL );`

Reply:
I'll revert this